sdk/servicebus/Azure.Messaging.ServiceBus/src/ServiceBusClient.cs

```diff
@@ -201,7 +201,7 @@ public ServiceBusReceiver GetReceiver(string queueName, ServiceBusReceiverOption
         /// <param name="topicName"></param>
         /// <param name="subscriptionName"></param>
         /// <returns></returns>
-        public ServiceBusReceiver GetReceiver(string topicName, string subscriptionName)
+        public ServiceBusReceiver GetSubscriptionReceiver(string topicName, string subscriptionName)
```

Comment:
As per the API feedback, can we change the queue receiver method name to `GetQueueReceiver()` or remove the "subscription" from this method?

Reply:
Same for the processor.

Reply:
I need to think about this a bit more since it will require shuffling around parameters for the session receiver methods. I will address this in a follow-up PR.